docs/standards/generic-standards/lsp14-ownable-2-step.md

```diff
@@ -0,0 +1,75 @@
+---
+sidebar_label: 'LSP14 - Ownable 2-Step'
+sidebar_position: 3
+---
+
+# LSP14 - Ownable 2-Step
+
+:::info Standard Document
+
+[LSP14 - Ownable 2-Step](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-14-Ownable2Step.md)
+
+:::
+
+## Introduction
+
+**LSP14 - Ownable2Step** is a modified version of [EIP173 - Contract Ownership Standard](https://eips.ethereum.org/EIPS/eip-173) that uses a safer mechanism for transferring and renouncing ownership.
```

Comment:
Please move this paragraph below under the following section. As follow:

---

## What does this standard represent ?

**LSP14 - Ownable2Step** is an extended version of [EIP173 - Contract Ownership Standard](https://eips.ethereum.org/EIPS/eip-173) that uses a safer mechanism for transferring and renouncing ownership. It modifies the processes of _tranferring and renouncing ownership_ in the following way: